src/dev/build/build_distributables.js

```diff
@@ -21,7 +22,8 @@ import {
   InstallDependenciesTask,
   OptimizeBuildTask,
   RemovePackageJsonDepsTask,
-  TranspileSourceTask,
+  TranspileBabelTask,
+  TranspileTypescriptTask,
```

Comment:
Yeah, I think `TypeScript` is correct in prose but IMO caps in camelCase should be based on words, os `TypeScript` would imply that the prose spelling is `Type Script` to me. I also think it makes more sense as `typescript` in snake_case, rather than `type_script`, for the same reason... So I'm going to stick with what I have.